Sources/Data.swift

```diff
@@ -28,7 +28,7 @@ extension Data: RangeReplaceableCollection, MutableCollection {
     }
 
     #if swift(>=3.0)
-        public mutating func replaceSubrange<C : Collection where C.Iterator.Element == Byte>(subRange: Range<Int>, with newElements: C) {
+        public mutating func replaceSubrange<C : Collection where C.Iterator.Element == Byte>(_ subRange: Range<Int>, with newElements: C) {
```

Comment:
I think this would fit Swift and the standard library better with the current snapshot as `replace(subRange: ..., with: )`, but that may be better suited for a different PR